modules/apps/headless/headless-builder/headless-builder-test/src/testIntegration/resources/com/liferay/headless/builder/resource/test/dependencies/expected_openapi.json

```diff
@@ -96,6 +106,48 @@
 					"name": "ListEntry"
 				}
 			},
+			"PageIndividualSchemaName": {
```

Comment:
This schema is not used. Can we delete it and not generate it?

Reply:
As I could see, and it is implemented. When a schema is selected in a Endpoint, the OpenAPI will contains both: Singular and Page one

Reply:
Yes but for the single element endpoint, it is not needed to create a page schema.